NEWS.rst

```diff
@@ -6,6 +6,9 @@ http://twistedmatrix.com/trac/ticket/<number>
 Twisted 17.9.0 (2017-09-23)
 ===========================
 
+This is the last Twisted release where Python 3.3 is supported, on any
```

Comment:
This looks like we are re-writing history here.
I think that we should not touch previous release notes.

So we should just update the release notes for the next (future) release and drop support after that...
but I am not familiar with the process of dropping a python version.

Reply:
When Python 2.6 support was dropped, the NEWS.rst file was modified after the fact to update this: http://twistedmatrix.com/trac/ticket/8651

Since that seemed to be OK, I did it here.

Dropping support for a Python release happens so infrequently, that I thought it was OK.